docs/sources/datasources/tempo/service-graph.md

```diff
@@ -100,9 +100,9 @@ Each circle's color represents the percentage of requests in each state:
 
 Service graph view displays a table of request rate, error rate, and duration metrics (RED) calculated from your incoming spans. It also includes a node graph view built from your spans.
 
-{{< figure src="/static/img/docs/tempo/apm-table.png" class="docs-image--no-shadow" max-width="500px" caption="Screenshot of the Service Graph view" >}}
+{{< figure src="/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-service-graph.png" class="docs-image--no-shadow" max-width="500px" caption="Screenshot of the Service Graph view" >}}
 
-For details, refer to the [Service Graph view documentation](/docs/tempo/latest/metrics-generator/service-graph-view/).
+For details, refer to the [Service Graph view documentation](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-service-graph.png).
```

Comment:
Yes, it should be a link. Your suggestion has the same mistake that I had with the image. I've updated the link here. 
```suggestion
For details, refer to the [Service Graph view documentation](/docs/tempo/<TEMPO_VERSION>/metrics-generator/service-graph-view/).
```